src/inmanta/export.py

```diff
@@ -532,6 +532,7 @@ def commit_resources(
         LOGGER.info("Sending resource updates to server")
         for res in resources:
             LOGGER.debug("  %s", res["id"])
+        LOGGER.debug("Parts:  %s", self._resource_sets)
```

Comment:
I'm not sure if "parts" is the best term here. Any reason not to go for just "resource sets"?

Reply:
not intended to commit, but it is useful, so I tried to improve it